### PR TITLE
Bono/goth 376

### PR DESCRIPTION
**Pull request by bonomite**
- created staff folder and staffslug.vue
- updated AuthorProfile.vue file with changing the name and adding a staffPage prop to render differently to match the design
- added load more button to tags page
- added section content page formatting to the tags page too

### CODE REVIEW

File: assets/scss/global.scss [hunk 1]
```diff
@@ -1,5 +1,14 @@
 $contentWidth: 1440px;
 $mobileBreakpoint: '<lg';
+$col-fixed-width-330: 330px;
```
Comment (bonomite) [inline assets/scss/global.scss:3]: moved these to global and made them specific.. and added it to a class for ease of use

File: components/ArticleFooter.vue [hunk 2]
```diff
@@ -33,7 +33,7 @@ const onTagClicked = (tag) => {
     <hr class="black mb-4 md:mb-6" />
     <div class="grid gutter-x-30">
       <div class="profile-col">
-        <article-footer-profile
+        <author-profile
```
Comment (bonomite) [inline components/ArticleFooter.vue:36]: changed the name to make it more universal and not tied to the article page

File: composables/data/articlePages.ts [hunk 3]
```diff
@@ -97,5 +97,7 @@ export function normalizeArticlePage(article: Record<string, any>): ArticlePage
 
 // Transform a list of article page data from the API into a simpler and typed format
 export function normalizeFindArticlePagesResponse(articlesResponse: any): ArticlePage[] {
+    //console.log(articlesResponse)
```
Comment (bonomite) [inline composables/data/articlePages.ts:100]: commented out the console... the rest is format noise

File: composables/types/Page.ts [hunk 4]
```diff
@@ -55,6 +55,12 @@ export interface ArticlePage extends Page {
 
 }
 
+export interface StaffPage extends Page {
```
Comment (bonomite) [inline composables/types/Page.ts:58]: I was mimicking what was done for the Tags page... but I am not using this.. and not sure if I do need it.

File: pages/[sectionSlug]/index.vue [hunk 5]
```diff
@@ -35,7 +35,7 @@ onMounted(() => {
         :tags="[
           {
             name: article.section.name,
-            slug: article.section.slug,
+            slug: `/tags/${article.section.slug}`,
```
Comment (bonomite) [inline pages/[sectionSlug]/index.vue:38]: making sure it is pointing to the root/tags/....

I did this in a new places that needed it.

**Review by kimlarocca — APPROVED** [review]

**Review by kimlarocca — APPROVED** [review]

**Review by kimlarocca — APPROVED** [review]

**Review by kimlarocca — APPROVED** [review]

**Pull request merged**